### DISCUSSION TITLE
Mac cfclient problem (intel chip): Crazyradio PA not showing up on client

### DISCUSSION THREAD

**Discussion by joebillips** [bitcraze/discussions#1101] [Q&A]
When I run cfclient it prints this out and never finds the crazyflie 2.1.
<img width="981" alt="Screenshot 2023-12-23 at 2 05 22 PM" src="https://github.com/bitcraze/discussions/assets/54544943/cc258790-bf84-49a9-9d78-2ae0c9ec1f1c">

When I try and "select an interface" the crazyradio PA doesn't show up:
<img width="1350" alt="Screenshot 2023-12-23 at 2 06 11 PM" src="https://github.com/bitcraze/discussions/assets/54544943/806303f9-8467-4a9f-9ce5-0bc18989fbf0">

But when I look at the devices connected to usb it shows the crazyradio PA:
<img width="509" alt="Screenshot 2023-12-23 at 2 07 11 PM" src="https://github.com/bitcraze/discussions/assets/54544943/044479ff-b1be-4665-b209-1be317a12255">

Cfclient is able to use the radio to cold boot the drone and flash with the preset firmware so I know the connection is working, it's just not showing up on the client.

I've tried running the cfclient in the bitcraze linux virtual machine and that had the exact same problem— able to cold boot but not connect.

Please help! :(((

**Comment by ataffanel** [comment]
The interface drop-down will only show information about detected Crazyflie, not Crazyradio. You need to click the scan button, the radio will light up mainly red, and any Crazyflie detected will be displayed in the drop down and available to connect.

If this does not work, it is likely your Crazyflie is not on the default address anymore (changed by a previous user for example). The address can be discovered and changed by connecting the Crazyflie over USB, clicking scan, and connecting the Crazyflie. This will allow you to enter the configuration menu via connect/cofigure 2.x.
